pkg/motor/motor.go

```diff
@@ -203,3 +232,42 @@ func (w *motorNodeImpl) AddCredentialVerificationMethod(id string, cred *did.Cre
 
 	return nil
 }
+
+func (m *motorNodeImpl) SendTokens(req mt.PaymentRequest) (*mt.PaymentResponse, error) {
+	// Build Message
+	fromAddr, err := types.AccAddressFromBech32(req.GetFrom())
+	if err != nil {
+		return nil, err
+	}
+
+	toAddr, err := types.AccAddressFromBech32(req.GetTo())
+	if err != nil {
+		return nil, err
+	}
+
+	amount := types.NewCoins(types.NewCoin("snr", types.NewInt(req.GetAmount())))
```

Comment:
we might want to check on 'Amount' from the request before attempting to send the transfer.

Reply:
Fixed @joshLong145